core/frame/frame.go

```diff
@@ -62,6 +60,8 @@ type HandshakeFrame struct {
 	AuthPayload string
 	// Version is used by the source/sfn to communicate their spec version to the server.
 	Version string
+	// Metadata stores additional data, it is an map[string]string{} that be encoded in msgpack.
+	Metadata []byte
```

Comment:
shall we explicit `Metadata` as ai meta?

Reply:
I changed the field name to `FunctionDefinition`.